### ISSUE TITLE
Gunbad left wing boss

### ISSUE THREAD

**Issue by Xicoz** [WarEmu/WarBugs#13121]
Expected behavior and actual behavior:
after wiping to the boss a couple times, now when we portal back into the boss room, he aggros instantly killing one of us before the whole grp can even get inside making it now impossible to complete the run.

Steps to reproduce the problem:

Screenshots/Videos or archive.org evidences:

**Comment by grunbag** [comment]
I need more info about how this is happening

**Comment by grunbag** [comment]
He reset full health during the fight or when the group died ?

**Comment by Xicoz** [comment]
After we all died, so he was resetting properly

**Comment by grunbag** [comment]
Ok thanks

**Comment by tog89** [comment]
My group today experienced similar issues.  Boss was fine on first fight but buggy behavior, including, aggro on party leader on zone into instance, even before party zoned in began after the first fight.  We wiped, waited until boss reset to full health and went to middle before releasing.  Would not reset to normal behavior 3 times out of five.  Also, one time it was normal, fight went to squig armor stage and party wipe occurred.  Every boss instance after that, the boss kept the damage defuff from the squig armor even though fight was reset.  Tried, zoning to capital and re-entering gunbad, tried reforming the party with new leader, no solution found.  Also, one party member would zone to Altdorf 20% of the  time.
